src/Std/Data/DTreeMap/Lemmas.lean

```diff
@@ -661,6 +678,19 @@ theorem getKey?_eq_some_getKey [TransCmp cmp] {a : α} {h'} :
     t.getKey? a = some (t.getKey a h') :=
   Impl.getKey?_eq_some_getKey t.wf
 
+theorem getKey_beq [TransCmp cmp] {k : α} (h' : k ∈ t) :
```

Comment:
Should this lemma get a different name in the tree map?

Reply:
Good point, I renamed it to `compare_getKey_self`, which I think better fits its statement.